databases/week2/QestionNo.3.txt

```diff
@@ -0,0 +1,22 @@
+/* Get all the tasks assigned to users whose email ends in @spotify.com*/
+
+	select * from  task
+	inner join user on task.status_id=user.id
+	where user.email like '%@spotify.com';
+
+/* Get all the tasks for 'Donald Duck' with status 'Not started*/
+
+	select  *from  task
+	right join user on task.status_id=user.id
```

Comment:
Here you are trying to join `task.status_id` with the `user.id` which is not correct. What do you need to do instead?